docs/reference/search-api.md

```diff
@@ -69,3 +70,40 @@ Search for documents matching a query in the given index `<index name>`.
 | **hits**             | Results of the query           | `[hit]` |
 | **numHits**         | Total number of matches        |  `number`  |
 | **numMicrosecs**    | Processing time of the query   |  `number`  |
+
+
+
+### Export search from an index
+
+```
+GET api/v1/indexes/<index name>/search/stream
+```
+
+Export documents matching a search query in the given index `<index name>`, in a specified output format.
+
+The output format can be one of the following:
+ -  [Comma Separated Values](https://datatracker.ietf.org/doc/html/rfc4180)
+ -  [ClickHouse Row Binary](https://clickhouse.tech/docs/en/interfaces/formats/#rowbinary)
+
+#### Path variable
+
+| Variable      | Description   |
+| ------------- | ------------- |
+| **index name** | The index name |
+
+
+#### Get parameters
+
+| Variable | Type | Description | Default value |
+|----------|------|-------------|---------------|
+| **query** | `String` | Query text. See the [query language doc](query-language.md) (mandatory) | |
+| **fastField** | `String` | Name of fast field to retrieve from documents (mandatory)| |
```

Comment:
It's the name of a field. And this field must be marked as "fast" in the config.